versions.cfg

```diff
@@ -46,6 +46,7 @@ mr.developer = 2.0.1
 nose = 1.3.7
 packaging = 21.3
 pathlib2 = 2.3.6
+# pip >= 22 no longer supports Python 3.6
 pip = 21.3.1
```

Comment:
This version pin actually hurts:

* If set to a higher version Python 3.6 breaks.
* If kept at the currently used version `pip` has to be pinned in `tox.ini` because otherwise the buildout run breaks because it is not able to downgrade to this `pip` version.

Could we try to remove this version pin?
(I tried it locally and it did not hurt.)

Reply:
Okay, let's try to remove it.

Reply:
Previously it did not hurt because `zc.buildout` did not use `pip`, bot version 3 does.